### PR TITLE
Document release process with diagrams

### PR DESCRIPTION
**Pull request by lavocatt**
To help a newcomer to understand our release process, adds a graphical documentation that aims to ease the learning curve. The graph is done using builtin mermaid

### CODE REVIEW

**Comment by ondrejbudai** [comment]
Can we do this with built-in Mermaid support? https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

**Comment by lavocatt** [comment]
> Can we do this with built-in Mermaid support? https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

Mermaid is not as good as plantuml, but that can do the job, since it's builtin, it so much easier.

File: README.md [hunk 1]
```diff
@@ -4,4 +4,59 @@ Maintainer Tools
...
+    end
+    RS2 -- Push Tag --> GH1
+    end
+    Release-. Asks for interactive release message edition .-> Foreperson
+    Foreperson -- Executes on local dev folder --> Release
+    subgraph Fedora [Fedora bot]
+        F1[Bot] -. waits for a new release .-> F1
+        F1 --> kerb([Get a kerberos ticket])
+        kerb --> fedpkg([schedule builds on koji])
+        fedpkg -- update --> bodhi[(Bohdi)]
```
[the remark this answers is not in the document]
Comment (lavocatt) [reply README.md:30]: Thanks!

**Review by achilleas-k — DISMISSED** [review]
This is awesome

**Comment by gicmo** [comment]
Have not looked at the PR itself, but it would super helpful if the PR description and also the commit message contained a a bit of a description; maybe not in this case but very often the why is more important then the what, which could be clear from the code itself.

**Comment by lavocatt** [comment]
> Have not looked at the PR itself, but it would super helpful if the PR description and also the commit message contained a a bit of a description; maybe not in this case but very often the why is more important then the what, which could be clear from the code itself.

I'll do next time.

**Review by ochosi — APPROVED** [review]
This is very nice!
However it will soon be a little outdated because the manual initial steps will be dropped and a GH action on a schedule will take over auto-tagging.
I set things up in a way to allow for manual tagging still though, so the workflow shown here won't be wrong, just incomplete.

File: README.md [hunk 2]
```diff
@@ -4,4 +4,63 @@ Maintainer Tools
...
+    Foreperson -- merge MR --> gitlab
+    Foreperson -- close false PR --> pagure
+    Foreperson -- merge PR --> pagure8
+    Foreperson -- Executes on local dev folder --> Release
+    linkStyle 0 stroke:red
+    linkStyle 1 stroke:red
+    linkStyle 2 stroke:red
+    linkStyle 3 stroke:red
+    subgraph Upstream
+    subgraph Release [Release script]
```
Comment (ochosi) [inline README.md:22]: @lavocatt  If you want to update this bit to the scheduled release model that'd be great!

**Comment by ochosi** [comment]
@lavocatt Much of the diagram is still accurate - mostly the fedora-bot bits would need updating.

**Pull request closed without merging**